modules/oe_subscriptions_anonymous/src/Form/AnonymousSubscribeForm.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_subscriptions_anonymous\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Mail\MailManagerInterface;
+use Drupal\Core\Render\RendererInterface;
+use Drupal\flag\FlagInterface;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * The form for the anonymous subscription.
+ */
+class AnonymousSubscribeForm extends FormBase {
+
+  /**
+   * Mail manager.
+   *
+   * @var \Drupal\Core\Mail\MailManagerInterface
+   */
+  protected MailManagerInterface $mailManager;
+
+  /**
+   * Configuration manager.
+   *
+   * @var Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * Renderer.
+   *
+   * @var Drupal\Core\Render\RendererInterface
+   */
+  protected $renderer;
+
+  /**
+   * Constructs a settings form.
+   *
+   * @param Drupal\Core\Mail\MailManagerInterface $mailManager
+   *   The mail manager.
+   * @param Drupal\Core\Config\ConfigFactoryInterface $configFactory
+   *   The config factory.
+   * @param Drupal\Core\Render\RendererInterface $renderer
+   *   The renderer.
+   */
+  public function __construct(
+    MailManagerInterface $mailManager,
+    ConfigFactoryInterface $configFactory,
+    RendererInterface $renderer) {
+    $this->mailManager = $mailManager;
+    $this->configFactory = $configFactory;
+    $this->renderer = $renderer;
+  }
```

Comment:
Same as above. I also missed that we don't need any of these services at all.

Reply:
Removed